esgf_utilities/esg_keystore_manager.py

```diff
@@ -28,15 +28,12 @@
 #   Keystore functions
 #------------------------------------
 
-def generate_tomcat_keystore(keystore_name, keystore_alias, private_key, public_cert, intermediate_certs):
+def generate_tomcat_keystore(keystore_name, keystore_alias, private_key, public_cert, cachain_file):
     '''The following helper function creates a new keystore for your tomcat installation'''
 
-    provider = "org.bouncycastle.jce.provider.BouncyCastleProvider"
+    # provider = "org.bouncycastle.jce.provider.BouncyCastleProvider"
```

Comment:
If this has been tested, I would suggest going ahead and deleting this line instead of leaving it commented.